test/Microsoft.ML.Tests/CalibratedModelParametersTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@eerhardt @yaeldekel I didn't know where to put these tests, so I created this file. If you think they should go elsewhere, please instruct me to do so.

Btw, I thought of putting them in the [CalibratorEstimators.cs](https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/TrainerEstimators/CalibratorEstimators.cs) tests, but it seems to me that file is more about testing the calibrator classes, whereas in here I am focusing in the CalibratedModelParameters classes.

Reply:
I think adding a new test file is reasonable.